pkg/utils/key.go

```diff
@@ -0,0 +1,70 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/spf13/pflag"
+)
+
+// ParseKey parse key by given format
+func ParseKey(flags *pflag.FlagSet, key string) ([]byte, error) {
+	switch flags.Lookup("format").Value.String() {
+	case "raw":
+		return []byte(key), nil
+	case "escaped":
+		return unescapedKey(key)
+	case "hex":
```

Comment:
For simplicity, can we just support hex only?

Reply:
maybe make hex as default format is enough?